main.cpp

```diff
@@ -275,296 +300,281 @@ struct Bakery
     void smellGreat();
 
     RyeBread breadToSell;
 };
 
-/*
-Thing 3) aqua park
-5 properties:
-    1) number of pools (int)
-    2) number of saunas (int)
-    3) total length of water slides (double)
-    4) opening hours (float)
-    5) ticket price (float)
-3 things it can do:
-    1) relax muscles
-    2) be fun
-    3) learning swimming
- */
+void Bakery::bakeBread(double bFlourAmount, RyeBread brd)
+{
+    std::cout << "Flour used: " << bFlourAmount << std::endl << "Timer set to: " << brd.bakingTime << std::endl;
+}
+
+float Bakery::cakeSell(std::string cakeName)
+{
+    if (cakeName == "Sweet Cake")
+    {
+        std::cout << "Product sold: " << cakeName << std::endl;
+        return 9430.f;
+    }
+    
+    return 0.f;
+}
+
+void Bakery::smellGreat()
+{
+}
 
 struct AquaPark
 {
-    //number of pools (int)
     int numPools = 5;
-    //number of saunas (int)
     int numSaunas = 3;
-    //total length of water slides (double)
     float totalWaterSlidesLength = 121.56f;
-    //opening hours (float)
     float openAt = 9.30f;
-    //ticket price (float)
     float regularTicker = 13.21f;
 
-    //relax muscles
     void relaxMuscles(bool goToSauna, int swimmingIntensity);
-    //be fun
     void haveFun(float totalWaterSlidesLength, int numberOfSlides);
-    //learning swimming
     bool learnToSwim(int age, int timeSpent);
 };
 
-/*
-Thing 4) house
-5 properties:
-    1) number of windows (int)
-    2) size of living room (float)
-    3) room height (float)
-    4) number of bathrooms (int)
-    5) total floor size (float)
-3 things it can do:
-    1) provide shelter
-    2) provide rest
-    3) get dirty
- */
+void AquaPark::relaxMuscles(bool goToSauna, int swimmingIntensity)
+{
+    goToSauna = false;
+    if (swimmingIntensity > 5)
+    {
+        goToSauna = true;
+        std::cout << "You should use one of the sauna's to relax your muscles!" << std::endl;
+    }
+}
+void AquaPark::haveFun(float aTotalWaterSlidesLength, int numberOfSlides)
+{
+    std::cout << "Total length of our " << numberOfSlides << " slides is: " << aTotalWaterSlidesLength << " which means that it's not possible to NOT have fun!" << std::endl;
+}
+bool AquaPark::learnToSwim(int age, int timeSpent)
+{
+    if (age < 10 && timeSpent > 60)
+        return true;
+    else if (age >= 10 && timeSpent > 120)
+        return true;
+
+    return false;
+}
 
 struct House
 {
-    //number of windows (int)
     int numWindows = 8;
-    //size of living room (float)
     float livingRoomSize = 31.8f;
-    //room height (float)
     float roomHeight = 2.8f;
-    //number of bathrooms (int)
     int numBathRooms = 3;
-    //total floor size (float)
     float totalFloorSize = 155.2f;
 
-    //provide shelter
     void provideShelter();
-    //provide rest
     void provideRest(float sleepQuality);
-    //get dirty
-    bool getDirty(int numPeople, int numAnimals, float totalFloorSize);
+    bool getDirty(int numPeople, int numAnimals, float hTotalFloorSize);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise